folsom/src/main/java/com/spotify/folsom/client/DefaultRawMemcacheClient.java

```diff
@@ -223,8 +223,8 @@ private DefaultRawMemcacheClient(
   public <T> CompletionStage<T> send(final Request<T> request) {
     if (request instanceof SetRequest) {
       SetRequest setRequest = (SetRequest) request;
-      byte[] value = setRequest.getValue();
-      if (value.length > maxSetLength) {
+      int margin = 40; // see MaxSetLengthTest
+      if (setRequest.size() > maxSetLength - margin) {
```

Comment:
By any chance the extra stuff in `setRequest.size()` contribute to the `margin`?

Reply:
What I'm more interested in is that, is there some constant value that we can use, such as `value.length > maxSetLength - another_margin`, so we don't have to calculate this `size()` every time? We don't need to reach the exact limit of memcached if we can have some constant here.